src/main/scala/it/pagopa/interop/authorizationmanagement/api/impl/ClientApiServiceImpl.scala

```diff
@@ -49,11 +47,9 @@ final case class ClientApiServiceImpl(
     val operationLabel: String = s"Creating client for Consumer ${clientSeed.consumerId}"
     logger.info(operationLabel)
 
-    val clientId = uuidSupplier.get()
-
-    val persistentClient       = PersistentClient.toPersistentClient(clientId, clientSeed)
+    val persistentClient       = PersistentClient.toPersistentClient(clientSeed)
```

Comment:
it's preferable to pass the `uuidSupplier` to `toPersistentClient` in order to test the uuid generation